src/main/java/nextstep/ladder/domain/Bridges.java

```diff
@@ -3,14 +3,19 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Random;
+import java.util.stream.IntStream;
 
 public class Bridges {
 
     private static final Random RANDOM = new Random();
+    private final List<Boolean> bridges;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가해보면 가독성이 좋아질 것 같아요.
```suggestion
    private static final Random RANDOM = new Random();
    
    private final List<Boolean> bridges;
```